src/Framework/App.Ref/src/CompatibilitySuppressions.xml

```diff
@@ -1,7 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Suppressions xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
   <Suppression>
     <DiagnosticId>PKV004</DiagnosticId>
     <Target>net7.0</Target>
   </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV004</DiagnosticId>
+    <Target>net8.0</Target>
```

Comment:
For my learning - what are these files for? And why do we need both `net7.0` and `net8.0` in them?

Reply:
I'm not 100% sure of the reasons but I ran this delta past @wtgodbe since it was automatically introduced when I ran the `GenerateProjectList` task after adding the new source generator project to the repo.

From my understanding, it's a requirement of the [package validation SDK](https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/overview) we use for validating changes across versions.

As to why we have both versions, I'm thinking we might be able to get away with just having `net8` in the main branch but I'll let @wtgodbe share.

For now, I just settled with whatever the build produced here 😄

Reply:
I think net7.0 can be removed now that the source is targeting .NET 8.

Reply:
> I think net7.0 can be removed now that the source is targeting .NET 8.

Roger that. It probably makes sense to do this in a separate PR. I assume there's no harm to having `net7.0` here since we've had it in the main branch for a while.

We might also want to add removing the older framework target as part of our [updating branding instructions](https://github.com/dotnet/aspnetcore/blob/main/docs/UpdatingMajorVersionAndTFM.md).

Reply:
Notice that these files keep changing because the order of the namespaces emitted into the file keep changing. Anyone know the reason for this non-determinism?

Reply:
@ViktorHofer and @smasher164 own the Package Validation SDK - any idea on why there's a net7.0 & and a net8.0 section in the generated file, or why there may nondeterminism in the ordering of the file? It's been somewhat noisy lately

Reply:
@smasher164 can you please take a look? We might have caused a non-deterministic behavior by ordering the compatibility differences in ApiCompat. Or maybe not and something else is wrong.